akd/tools/vector_db_tool.py

```diff
@@ -0,0 +1,149 @@
+import os
+from typing import List, Optional
+
+import chromadb
+import chromadb.utils.embedding_functions as embedding_functions
+from langchain_core.documents import Document
```

Comment:
Is Langchain's `Document'` dependency actually needed? Can find another way? Like `akd.structures.Document` or something with similar but minimal data structure?

Reply:
I removed the Document dependency, we just need to pass a list of ids, texts and metadata, as expected by Chroma's `.add()` [method](https://docs.trychroma.com/docs/collections/add-data)